Admin/Admin.php

```diff
@@ -2926,7 +2926,7 @@ public function getAccessMapping()
     /**
      * {@inheritdoc}
      */
-    public function checkAccess($action, $object = null)
+    public function getAccess()
```

Comment:
@core23 I'm not sure. For example, I want to use this method in SonataDoctrineORMAdminBundle instead this code :
` {% set route_role = route_name | upper %}`
 ` {% if field_description.hasAssociationAdmin and field_description.associationadmin.hasRoute(route_name) and field_description.associationadmin.isGranted(route_role, value) %}
      `

If it will be protected we can't call this method there, but if we can solve this problem without this method we can make this method protected.